templates/base.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% block title %}
+        <title>Taxi Service</title>
+    {% endblock %}
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
+    {% block sidebar %}
+    {% endblock %}
+    {% block content %}
+        {% include "includes/content.html"%}
```

Comment:
In task, "override block content" means you should use template inheritance, not include. In this case it's ok, but consider it next time..